pkg/cmd/pulumi-language-java/main.go

```diff
@@ -966,4 +965,4 @@ type POMProject struct {
 	Artifact string `xml:"artifactId"`
 	// The version of the artifact produced by the project, encoded in the `<version>` tag.
 	Version string `xml:"version"`
-}
+}
```

Comment:
Accidentally removed newline?

Reply:
Yep and the linter caught it. *Grumbles*